Assets/Scripts/2D/MapScript.cs

```diff
@@ -210,6 +212,7 @@ public void PointerDown(BaseEventData data)
             if (!Manager.ViewingGlobe && !_isDraggingMap)
             {
                 Manager.ActivateEditorBrush(true);
+                ActivatedBrush.Invoke(true);
```

Comment:
What does this do?

Reply:
ActivatedBrush is an Unity event that is visible within the Unity3D editor. it's useful if you want to link events to listeners within the editor without having to add extra lines of code. There's one big donwside in the fact that this hides dependencies from the IDE/compiler.

In this case I'm using this event to control the states of the 'Drainage Regeneration', 'Exit Editor Mode' and 'Save' buttons in the UI